Floofbot/Services/RaidProtectionService.cs

```diff
@@ -61,6 +60,15 @@ public RaidProtectionService()
             userJoinsDelay = raidConfig["UserJoinsDelay"];
             maxNumberEmojis = raidConfig["MaxNumberEmojis"];
             maxNumberSequentialCharacters = raidConfig["MaxNumberSequentialCharacters"];
+            durationBetweenMessages = raidConfig["DurationBetweenMessages"];
+            maxMessageSpam = raidConfig["MaxMessageSpam"];
+        }
+        private async Task HandleBadMessage(SocketUser user, SocketMessage msg)
```

Comment:
Looks like this isn't being called anywhere (I'm assuming this is supposed to be used in place of R161?)

Reply:
Removed